components/tyk-mdcb/templates/deployment-mdcb.yaml

```diff
@@ -17,10 +17,11 @@ spec:
       {{- include "tyk-mdcb.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.mdcb.podAnnotations }}
       annotations:
+      {{- with .Values.mdcb.podAnnotations }}
```

Comment:
why do we need to move this statement under annotations? when I tested the current structure, it was working fine unless I missed something

Reply:
I have added a new annotation to watch for secrets. That's why this needs to be moved.